index.html

```diff
@@ -64,6 +64,159 @@ <h5> LET's CONNECT</h5>
+       </div>
+    <h1>Multi-Post Stories</h1>
+    <ul class = "work-ul">
+    <li><h2>CANOPY</h2></li>
+    
+    <li><h3><img src="./img/workimg/Counter.svg"> &nbsp; &nbsp;Back End Dev &nbsp; &nbsp; <img src="./img/workimg/Counter.svg"> &nbsp; &nbsp;2015 </h3></li>
+</ul>
+
+<p> A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+<ul class = "work-tags">
+    <li class="html">html</li>
+    <li class = "css">css</li>
+    <li class = "javascript">javascript</li>
+</ul>
+
+<button class="button button2">See Project</button>
+    </div>
+
+
+    
+    <div class="items work3">
+        <div class = "workimg"> <img src="./img/workimg/work3.svg" alt = "Work Image Screenshot1">
+       </div>
+    <h1>Tonic</h1>
+    <ul class = "work-ul">
+    <li><h2>CANOPY</h2></li>
+    
+    <li><h3><img src="./img/workimg/Counter.svg"> &nbsp; &nbsp;Back End Dev &nbsp; &nbsp; <img src="./img/workimg/Counter.svg"> &nbsp; &nbsp;2015 </h3></li>
+</ul>
+
+<p> A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+<ul class = "work-tags">
+    <li class="html">html</li>
+    <li class = "css">css</li>
+    <li class = "javascript">javascript</li>
+</ul>
+
+<button class="button button3">See Project</button>
+    </div>
+
+
+
+    <div class="items work4">
+        <div class = "workimg"> <img src="./img/workimg/work4.svg" alt = "Work Image Screenshot2">
+       </div>
+    <h1>Multi-Post Stories</h1>
+    <ul class = "work-ul">
+    <li><h2>CANOPY</h2></li>
+    
+    <li><h3><img src="./img/workimg/Counter.svg"> &nbsp; &nbsp;Back End Dev &nbsp; &nbsp; <img src="./img/workimg/Counter.svg"> &nbsp; &nbsp;2015 </h3></li>
+</ul>
+
+<p> A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+<ul class = "work-tags">
+    <li class="html">html</li>
+    <li class = "css">css</li>
+    <li class = "javascript">javascript</li>
+</ul>
+
+<button class="button button4">See Project</button>
+    </div>
+   
+</div>
+
+<div class="about-myself">
+    <div class="about-bg">
+    <h1>         About Myself</h1>
+        <div class = "about-text"> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
+
+        <h2> LET's CONNECT</h2>
+<div class="social-links">
+<ul>
+    <li ></li><a href="https://twitter.com/dheerajarya"><img src="./img/twitter.svg"></a>
+    </li>
+    <li ></li><a href="https://www.linkedin.com/in/dheeraj-sachdeva-502b2b8/"><img src="./img/Linkedin icon.svg"></a>
+</li>
+<li ></li><a href="#"><img src="./img/M.svg"></a>
+</li>
+<li ></li><a href="https://github.com/dheerajsachdeva"><img src="./img/github.svg"></a>
+    </li>
+    <li ></li><a href="#"><img src="./img/victory.svg"></a>
+    </li>
+    
+</ul>
+</div>
+<button class="button button5">Get my resume</button>
```

Comment:
It seems the button is yet to have the interaction as previously requested and I suggest you implement them. This is very important as it helps a user understand the various states a button might be in (either disabled, hovered, or pressed), and having this feature in your portfolio would provide a better UX to the user.

There are some CSS pseudo-classes that can be used to achieve that (`button:hover` (to style the state when the cursor hover upon the button), `button:active` (to style the pressed tate), and `button:disabled` (to style disabled state)).

[check this article out](https://www.digitalocean.com/community/tutorials/how-to-use-links-and-buttons-with-state-pseudo-classes-in-css)

You will be applying these colors on the various state and please do not forget to leave the button first in its enabled state instead of disabled according to the Figma template.

YOUR CURRENT DESIGN

![no hover](https://user-images.githubusercontent.com/84930355/212463659-c9d6f81e-cc4d-4d46-82b4-6e38397ffa7b.PNG)


FIGMA SPECIFICATION

![image](https://user-images.githubusercontent.com/106119105/212442800-37829ff9-3d90-4301-88b1-15588e0cb7c5.png)